frontend/src/main/components/Commons/ManageCowsModal.js

```diff
@@ -62,7 +62,7 @@ const ManageCowsModal = ({number, setNumber, isOpen, onClose, message, userCommo
       </Modal.Body>
       <Modal.Footer data-testid={"buy-sell-cow-modal-footer"} style={{borderTop: '0px'}}>
         <Button data-testid={"buy-sell-cow-modal-cancel"} variant="secondary" onClick={handleClose}>Cancel</Button>
-        <Button data-testid={"buy-sell-cow-modal-submit"} variant="primary" onClick={handleSubmit}>{message}</Button>
+        <Button data-testid={"buy-sell-cow-modal-submit"} variant="primary" onClick={handleSubmit}>{message[0].toUpperCase() + message.slice(1)}</Button>
```

Comment:
Well, you could do it this way, but it seems like overkill...

See comment below...

Reply:
resolving because of this:
https://github.com/ucsb-cs156-f24/proj-happycows-f24-10/pull/14#issuecomment-2480151773